beacon-chain/sync/pending_blocks_queue.go

```diff
@@ -109,6 +114,39 @@ func (r *RegularSync) sortedPendingSlots() []int {
 		slots = append(slots, int(s))
 	}
 	sort.Ints(slots)
-
 	return slots
 }
+
+func (r *RegularSync) validatePendingSlots() error {
```

Comment:
Add a comment for `validatePendingSlots`, hard to tell what this does just from looking at the name